ntc_templates/templates/zyxel_os_cfg_intf_group_get.textfsm

```diff
@@ -0,0 +1,16 @@
+Value INDEX (\d+)
+Value NAME (.+?)
+Value WAN_INTF (Any\sWAN|((WWAN|ADSL|VDSL),?)+)
+Value INTERFACE (.+(?<!\s))
```

Comment:
Not sure what this is looking for, can you add an example to the tests?

Reply:
Yip, I haven't spotted that one.
I corrected and added more examples to the tests